src/docs/blog/2025-10-07-day-5-move-to-normal-ward.md

```diff
@@ -0,0 +1,52 @@
+
+## The Move Across the Hall
+
+No sooner had I finished my breakfast than a team of nurses arrived to move me to my new room. The journey wasn't far - all the way directly across the hall!
+
+The transition brought several welcome changes:
+
+* **Oxygen line removed**: No longer needed, confirming my lungs were recovering well
+* **Heart monitor upgrade**: The continual monitoring equipment shrank to a portable Phillips MX40-2C4 that I can carry around my neck. It still sends all the data to the central nurse station, but I'm no longer tethered to a bedside machine
+* **Mobile pacemaker**: I still have to carry this around as well, though they mentioned it would be removed in the next day or two
+
+## The Temporary Pacemaker
+
+The mobile pacemaker is a slightly weird experience. The cables run inside me to somewhere close to my heart - you can find information about what colour wires do what if you're curious. The device allows the medical team to speed up or slow down my heart if needed, though to my knowledge it hasn't been used yet. Having those cables inside is an odd sensation, but it's temporary and an important safety measure during recovery.
+
+## Life in the Normal Ward
+
+Now I'm in a nice quiet room alone. No more beeping machines, though there are still plenty of nurse visits, tests, and medicine to take. The quieter environment is a welcome change after the hectic intermediate care ward.
+
+## Physiotherapy and Breathing Exercises
+
+The physio took me for a walk down the corridor, which went well. My mobility is improving daily.
+
+She also gave me some more advanced breathing exercises to do. These are crucial because of the discomfort and reduced activity - the temptation is to breathe superficially, which is bad for recovery. Shallow breathing allows bad stuff to build up, leading to coughing (which rates 10 out of 10 on the pain scale, even in the special braced position) and worse possibilities like infection if I don't do the breathing exercises properly and keep my airways clear.
+
+Despite the pain, I'm committed to doing the breathing exercises regularly. They're an essential part of preventing complications.
+
+## Progress Summary
+
+Day 5 marks a significant milestone in my recovery:
+
+* ✅ Moved to normal ward
+* ✅ Oxygen support no longer needed
+* ✅ Portable monitoring equipment
+* ✅ Increased mobility with corridor walks
+* ✅ Advanced breathing exercises
+* ⏳ Temporary pacemaker still in place (removal soon)
+
+The steady progression from intensive care to intermediate care and now to the normal ward shows good recovery. Each day brings more independence and fewer machines, though the vigilant medical monitoring continues to ensure everything heals properly.
```

Comment:
Small addition, I had got myself into and it off bed on a couple of occasions, but i don't recommend that, going to work with the nurses for a few more days to at least get me sat upright, then stand up myself, getting in and out of the chair using the brace pain is fine now, bed still maybe a bit much with the need to spin around as well as get upright or maybe better use of the bed controls are needed so the bed is in the easiest pain to get out, I forgot that important step.